src/undocommand.js

```diff
@@ -0,0 +1,200 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+import Command from '../command/command.js';
+
+/**
+ * Undo command stores batches in itself and is able to and apply reverted versions of them on the document.
+ *
+ * undo.UndoCommand
+ */
+export default class UndoCommand extends Command {
+	/**
+	 * @see engine.command.Command
```

Comment:
The whole comment isn't needed (hopefully) if you extend another class and overrides one of existing methods.